### ISSUE TITLE
Unable to resolve 'Openchain.MongoDb (>= 0.1.0-alpha1)

### ISSUE THREAD

**Issue by bathie007** [openchain/mongodb-storage#2]
Hi, 
trying to run openchain with mongodb, got an issue:
Unable to resolve 'Openchain.MongoDb (>= 0.1.0-alpha1)' for 'DNX,Version=v4.5.1'

**Comment by maurodelazeri** [comment]
same

**Comment by karneaud** [comment]
+1

**Comment by karneaud** [comment]
@bathie007 @maurodelazeri have you guys ever figured it out?

**Comment by maurodelazeri** [comment]
@karneaud I did not try fix it, but i think is the only way

**Comment by karneaud** [comment]
@maurodelazeri  you should try multi chain.... much more reliable

**Comment by maurodelazeri** [comment]
@karneaud are u using it? idk much about MultiChain yet... i ll check it out

**Comment by Flavien** [comment]
The MongoDB module is a community contribution and is maintained by @fluce. ~~It hasn't been updated to run with Openchain 0.7.~~

~~At the moment, I recommend using either the Sqlite storage or the SQL Server storage.~~

**Comment by Flavien** [comment]
I've updated the package to work with the latest version.

You can pull the following package from NuGet: https://www.nuget.org/packages/Openchain.MongoDb/0.2.0

**Comment by karneaud** [comment]
@Flavien yes. thanks for this, but i'm a bit reluctant now to use openchain based on the longevity of this project. It is most definitely easier to implement than multichain( given that i had to overcome the learning curve with both frameworks) but the maintainability appears to be unreliable. what is the future like for openchain...? Thanks for updating this package

**Comment by Flavien** [comment]
What do you mean by longevity? It's open source, anyone can submit pull requests for bugs they find or even fork the code and modify it to their needs.

Multichain and Openchain are actually very different: Openchain is open source, Multichain isn't.
